### PR TITLE
fix: analytics test

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR:
- move some files around
- fix cypress tests for analytics
- fix code smells

### CODE REVIEW

File: cypress/e2e/analytics/chartsHeader.cy.ts [hunk 1]
```diff
@@ -64,8 +148,9 @@ describe('Select platform view ', () => {
 describe('Select users', () => {
   beforeEach(() => {
     cy.setUpApi({});
+    setupIntercepts();
   });
-  it('values of user select should be maintained when navigating within different routes', () => {
+  it.only('values of user select should be maintained when navigating within different routes', () => {
```
Comment (pyphilia) [inline cypress/e2e/analytics/chartsHeader.cy.ts:153]: Remove.

File: src/locales/en/account.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   "CROP_IMAGE_MODAL_UNEXPECTED_ERROR": "An unexpected error occurred when trying to crop the image, please try again later",
   "MAIN_MENU": {
     "HOME": "Home",
+    "STATS": "Statistics",
```
Comment (pyphilia) [inline src/locales/en/account.json:13]: Maybe we want a clearer key?

File: src/modules/analytics/charts/TotalActionsByVerbChart.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    const formattedAggregateData = aggregateData.map((d) => ({
+      actionCount: (d.aggregateResult / totalActions) * 100,
+      type: d.actionType,
+    }));
+
+    // formattedAggregateData.forEach((d) => {
+    //   d.actionCount = parseFloat(
+    //     ((d.actionCount / totalActions) * 100).toFixed(2),
+    //   );
+    // });
```
Comment (pyphilia) [inline src/modules/analytics/charts/TotalActionsByVerbChart.tsx:74]: Remove?

**Review by pyphilia — APPROVED** [review]
LGTM! 📶

File: src/locales/en/account.json (hunk 2 above)
Comment (spaenleh) [reply src/locales/en/account.json:13]: Do you mean change to
```suggestion
    "STATISTICS": "Statistics",
```

or 
```suggestion
    "STATISTICS_MAIN_MENU": "Statistics",
```
but it is already contextual no ? So I think short key is ok. I can use `STATISTICS` if you prefer.

**Pull request merged**